content/app/github/commit-page-helper.js

```diff
@@ -47,10 +47,25 @@ define("content/app/github/commit-page-helper", function(require, exports, modul
 				return file;
 			},
 
+			/**
+			 * Find ranges of provided text in all file DOM elements.
+			 * @param  {String} text
+			 * @return {Range}
+			 */
+			findTextRangesInFiles : function (text) {
+				var files = CommitPageHelper.getAllFiles();
+				var rangesArrays = files.map(function (file) {
+					var data = file.getFileData();
+					return data.findTextRanges(text);
+				});
+				return Array.prototype.concat.apply([], rangesArrays);
```

Comment:
Ranges arrays is matrix. That's why I called I added "s" at the end of both words. And I need all ranges in one array.
This is the only way I found to do that in one line.
I will retype this to use for loop. It'll be more readable and I won't have matrix.